luaui/configs/hotkeys/grid_keys.txt

```diff
@@ -122,7 +122,8 @@ bind               sc_`  drawinmap
 
 bind          Ctrl+sc_e  select AllMap++_ClearSelection_SelectAll+
 bind           Ctrl+tab  select AllMap+_Builder_Idle+_ClearSelection_SelectOne+
-bind                tab  selectcomm
+bind          Shift+tab  selectcomm append
```

Comment:
appending seems new

Reply:
Yes, all 3 args are new. In my experience players have need for different usage often.

Reply:
What i mean is that shift+tab was not bound to any variante of com selection previously. This is a new binding

Reply:
It was, it was just either non functional or functional but hidden by the shift monkey check in previous

Reply:
it wasn't bound to any+tab tho, just tab, so shouldnt have triggered with shift

Reply:
Yeah, let's see what author intends

Reply:
I'm pretty sure the behavior is intended, just missing in the first iteration.

Reply:
idk, i assume the recently added implementation handled shift because it was just a copypaste from zk. the actual precedent are the pre-existing tab/ctrl+c bindings using uikeys, that have been there for years without shift

Reply:
I have no strong opinions, whatever allows better functionality is best option.  My original need was to solve the outdated exclusion check, and the way engine added to explicitly allow that, was derailed down the customparam rabbithole with a lot of raiding from ZK to achieve that, which likely has artefacts from the process. I originally had no intention of doing anything like this.

Reply:
I like the added consistency, but also I think it may be good to leave room for commander-specific actions (since the unit is so important) and mod+Tab would be a natural place to reserve for those.

Reply:
Well, these are defaults right, if the user wants to customize they are free to do what they want

Reply:
> Yeah, let's see what author intends

> was just a copypaste from zk

mfw you are the author

Reply:
I am resolving this, if you still have any objection feel free to comment and I'll reopen.